internal_cert_management/.env.example

```
@@ -14,12 +14,28 @@ SUBDOMAINS_RSA2048="udm.newyork.example.com udm.wisconsin.example.com"
 HOSTLIST_VAULT=udm.newyork.example.com
 SUBDOMAINS_VAULT="newyork.example.com wisconsin.example.com miyagi.example.com"
 CERT_TYPES_VAULT="rsa2048 ec256"
+# Subdomain matching the Vault host's own TLS cert (used by push_from_files.sh)
+# HOSTLIST_VAULT must be the Vault server's inventory hostname (not a UDM or API endpoint)
+# push_from_files.sh requires passwordless sudo on the Vault host, or set ANSIBLE_BECOME_PASS
+SUBDOMAIN_VAULT_HOST=newyork.example.com
+CERT_TYPE_VAULT_HOST=ec256
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CERT_TYPE_VAULT_HOST key should go before the HOSTLIST_VAULT key

(UnorderedKey)